modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/services/liferay/object/mdf-requests/useGetMDFRequests.ts

```diff
@@ -17,10 +17,15 @@ import {LiferayAPIs} from '../../common/enums/apis';
 import LiferayItems from '../../common/interfaces/liferayItems';
 import liferayFetcher from '../../common/utils/fetcher';
 
-export default function useGetMDFRequests() {
+interface IProps {
+	page: number;
+	pageSize: number;
+}
+
+export default function useGetMDFRequests({page, pageSize}: IProps) {
```

Comment:
```suggestion
export default function useGetMDFRequests(page: number, pageSize: number) {
```

Para funçoes nao precisamos usar as properties com chaves.